src/senaite/core/setuphandlers.py

```diff
@@ -105,62 +112,65 @@ def getNonInstallableProfiles(self):
-    ("Method", ["senaite_catalog_setup", "portal_catalog"]),
-    ("Multifile", ["senaite_catalog_setup", "portal_catalog"]),
-    ("Preservation", ["senaite_catalog_setup", "portal_catalog"]),
-    ("ReferenceDefinition", ["senaite_catalog_setup", "portal_catalog"]),
-    ("ReferenceSample", ["senaite_catalog", "portal_catalog"]),
-    ("SampleCondition", ["senaite_catalog_setup", "portal_catalog"]),
-    ("SampleMatrix", ["senaite_catalog_setup", "portal_catalog"]),
-    ("SamplePoint", ["senaite_catalog_setup", "portal_catalog"]),
-    ("SampleType", ["senaite_catalog_setup", "portal_catalog"]),
-    ("SamplingDeviation", ["senaite_catalog_setup", "portal_catalog"]),
-    ("StorageLocation", ["senaite_catalog_setup", "portal_catalog"]),
-    ("SubGroup", ["senaite_catalog_setup", "portal_catalog"]),
-    ("Supplier", ["senaite_catalog_setup", "portal_catalog"]),
-    ("WorksheetTemplate", ["senaite_catalog_setup", "portal_catalog"]),
+    ("ARReport", [REPORT_CATALOG, PORTAL_CATALOG]),
+    ("ARTemplate", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("ARTemplate", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Analysis", [ANALYSIS_CATALOG]),
+    ("AnalysisCategory", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("AnalysisProfile", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("AnalysisRequest", [SAMPLE_CATALOG]),
+    ("AnalysisService", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("AnalysisSpec", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Attachment", [SENAITE_CATALOG, PORTAL_CATALOG]),
+    ("AttachmentType", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Batch", [SENAITE_CATALOG, PORTAL_CATALOG]),
+    ("BatchLabel", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Calculation", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Container", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("ContainerType", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Department", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Instrument", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("InstrumentType", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("LabContact", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("LabProduct", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Manufacturer", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Method", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Multifile", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Preservation", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("ReferenceDefinition", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("ReferenceSample", [SENAITE_CATALOG, PORTAL_CATALOG]),
+    ("SampleCondition", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("SampleMatrix", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("SamplePoint", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("SampleType", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("SamplingDeviation", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("StorageLocation", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("SubGroup", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("Supplier", [SETUP_CATALOG, PORTAL_CATALOG]),
+    ("WorksheetTemplate", [SETUP_CATALOG, PORTAL_CATALOG]),
```

Comment:
Some content types are missing, like `Worksheet`, `DuplicateAnalysis`, `ReferenceAnalysis`, `Contact`, `InstrumentCalibration`, etc.

Reply:
The other types were added by the setuphandler `setup_core_catalogs` which maps the types from here: https://github.com/senaite/senaite.core/blob/9adcd931ecec4c9f8cef26cfd0117dda71dc4dff/src/senaite/core/catalog/analysis_catalog.py#L73-L78

However, for completeness I added the missing types in  9adcd931ecec4c9f8cef26cfd0117dda71dc4dff from here:
https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/__init__.py#L57-L121